sky/core.py

```diff
@@ -208,6 +208,8 @@ def _start(
                                stream_logs=True,
                                cluster_name=cluster_name,
                                retry_until_up=retry_until_up)
+    storage_mounts = backend.get_cluster_metadata_storage_mounts(handle)
+    backend.sync_file_mounts(handle, None, storage_mounts)
```

Comment:
This will also trigger our file mount syncing code. This can be problematic - e.g., if a user's local path has been deleted, this will likely raise an error and prevent the user from starting a stopped cluster. 

Since rsync file_mounts and `COPY` mode objects would anyway be persistent on the remote disk volume, is there a way we can execute only `MOUNT` mode file_mounts and skip the local->remote upload for them, if any?

Reply:
It seems we're specifically executing in `MOUNT` mode only here, as we pass `None` as an argument to `backend.sync_file_mounts()`. When `file_mounts` parameter is set to `None`, `_execute_file_mounts()` will return immediately without taking any actions. Please correct me if I'm overlooking anything with this appraoch!